nvim/init.lua

```diff
@@ -557,6 +759,59 @@ require('lazy').setup({
     end
   },
 
+  -- auto save
+  {
+    "okuuva/auto-save.nvim",
+    cmd = "ASToggle", -- optional for lazy loading on command
+    event = { "InsertLeave", "TextChanged" }, -- optional for lazy loading on trigger events
+    opts = {
+      execution_message = {
+        enabled = false,
+      },
+      debounce_delay = 5000,
+    },
+  },
+
+  {
+    "saccarosium/nvim-whitespaces",
```

Comment:
This is a job for formatters I think

Reply:
Sometimes we may not have a formatter for a file type. This is a minimum safeguard to avoid us from mistakes. It’s happened with me when editing json where I didn’t have a firmatter since I’m not editing JSON too often. The same can be applicable for yaml. The point to have some must automation for such a basic but problematic thing which is easily to happen when editing with neovim

Reply:
I agree with this